ui/src/utils/utils.js

```diff
@@ -64,10 +64,10 @@ export const printTimeDiff = (took) =>
 	Math.floor((took || 0) / 60)
 	.toString()
 	.padStart(2, '0') +
```

Comment:
Should the Duration be 
1m 29s
instead of 
01m 29s

Reply:
Done - reformatted duration to '1m 29s'

![image](https://user-images.githubusercontent.com/67776356/120970279-8f755380-c7ae-11eb-9530-e13fc6d1aa5e.png)

**Figure: Reformatted duration**